src/services/pocket-relayer.ts

```diff
@@ -680,7 +692,17 @@ export class PocketRelayer {
       }
     }
 
-    const node = await this.cherryPicker.cherryPickNode(application, nodes, blockchainID, requestID)
+    let node: Node
+
+    // Before cherry picking, check to see if preferred node is in the set of good nodes
+    const preferredNodeIndex = nodes.findIndex((x) => x.address === preferredNodeAddress)
+
+    if (preferredNodeAddress && preferredNodeIndex > 0) {
+      console.log('STICKINESS SUCCESS', preferredNodeAddress)
```

Comment:
if these console.logs are meant to be kept then should be changed to `logger.log`, otherwise removed

Reply:
Ah yea I did want to see them on staging/canary at first, I will change them.

Reply:
as the way it works now, you won't be able to see them on DD unless you do a logger.log